SOURCES/etc/xapi.d/plugins/smartctl.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import json
+import sys
+import subprocess
+import XenAPIPlugin
+
+sys.path.append('.')
+from xcpngutils import configure_logging, run_command, error_wrapped
+from xcpngutils.operationlocker import OperationLocker
+
+_LOGGER = configure_logging('smartctl')
+
+def list_disks():
+    disks = []
+    try:
+        output = subprocess.check_output(['smartctl', '--scan'])
```

Comment:
Also other plugins are using `run_command` from **xcpngutils**. Can you use it. It is easy to mock in the tests.